fastlane/swift/Actions.swift

```diff
@@ -0,0 +1,11 @@
+// This class is automatically included in FastlaneRunner during build
+
+// This autogenerated file will be overwritten or replaced when running "fastlane generate_swift"
+//
+//  ** NOTE **
+//  This file is provided by fastlane and WILL be overwritten in future updates
+//  If you want to add extra functionality to this project, create a new file in a
+//  new group so that it won't be marked for upgrade
+//
+
+import Foundation
```

Comment:
I was trying to add a new action:

```
$ bundle exec fastlane new_action swiftSayHello
...
$ bundle exec fastlane generate_swift
```

and this is how the `Actions.swift` looks like:

```
import Foundation
func swift(apiToken: String,
           development: Bool = false) {
  let command = RubyCommand(commandID: "", methodName: "swift", className: nil, args: [RubyCommand.Argument(name: "api_token", value: apiToken),
                                                                                       RubyCommand.Argument(name: "development", value: development)])
  _ = runner.executeCommand(command)
}
func swiftSayHello(apiToken: String,
                   development: Bool = false) {
  let command = RubyCommand(commandID: "", methodName: "swift_say_hello", className: nil, args: [RubyCommand.Argument(name: "api_token", value: apiToken),
                                                                                                 RubyCommand.Argument(name: "development", value: development)])
  _ = runner.executeCommand(command)
}
```

1. Not sure if the default `args` are okay: `api_token` and `development`. 
2. Why is the `swift` method generated? 🤔

Reply:
I wonder if this was due to the way you named your action? 🤔 Actions are usually named like `swift_say_hello` so I'm wondering if there is an issue deeper down somewhere in that logic 🙃 I will look into

Reply:
There was a prompt and I typed 👇 

```
Name of your action: swift_say_hello
```

Reply:
> Not sure if the default args are okay: api_token and development.

They are just default params. The equivalent ruby action is created `fastlane/actions/swift_say_hello.rb` and you should edit `available_options` there. Once you're done with that, just rerun `fastlane generate_swift` to regenerate `Actions.swift` file to match the ones you just updated in the ruby file.

> Why is the swift method generated? 🤔

It looks like I had a mess with the `fastlane/actions/...` catalog and there was `swift_action.rb`, so all good 👍 

Thank you @joshdholtz for explaining it! 🙇